test/e2e/mixedcpus/mixedcpus.go

```diff
@@ -19,6 +19,7 @@ package mixedcpus
 import (
 	"context"
 	"fmt"
+	"github.com/openshift-kni/mixed-cpu-node-plugin/test/e2e/fixture"
```

Comment:
nit: out of order imports

Reply:
I'll fix it in another PR if you don't mind. I would like to merge if there's nothing critical because I would like to start working on the NTO side on Sunday

Reply:
no problem